macros/split_to_table_values.sql

```diff
@@ -0,0 +1,18 @@
+{%- macro split_to_table_values(table_array) -%}
+    {# Used in conjunction with split_to_table, this macro returns the split_to_table
+        values associated with the split_to_table function
+       ARGS:
+         - table_array (string) the table array form of the split_column.
+       RETURNS: A new column containing the split data.
+    #}
+    {%- if target.type ==  'postgres' -%} 
```

Comment:
You could potentially merge these 2 into `if target.type in ('bigquery','postgres',)` to consolidate code